src/lib/utils/mapSuctionTensionToStatus/index.ts

```diff
@@ -0,0 +1,44 @@
+export type WaterSupplyLabelType = 'Gut' | 'Mäßig' | 'Kritisch'
+
+export interface WaterSupplyStatusType {
+  suctionTensionRange: [number, number]
+  label: WaterSupplyLabelType
+  id: 'good' | 'medium' | 'critical'
+}
+
+export const WATER_SUPPLY_STATUSES: WaterSupplyStatusType[] = [
+  {
+    suctionTensionRange: [0, 25],
+    label: 'Gut',
+    id: 'good',
+  },
+  {
+    suctionTensionRange: [25, 62],
+    label: 'Mäßig',
+    id: 'medium',
+  },
+  {
+    suctionTensionRange: [62, 250],
+    label: 'Kritisch',
+    id: 'critical',
+  },
+]
+
+/**
+ * Maps a real suction tension ("Saugspannung") value (usually between 0 and 240)
```

Comment:
Great new habit! I love it!

Reply:
Yes, it's super helpful. I wanna get a bit more into the syntax. You could even describe examples etc.